ansible_collections/arista/cvp/galaxy.yml

```diff
@@ -7,7 +7,7 @@ namespace: arista
 name: cvp
 
 # The version of the collection. Must be compatible with semantic versioning
-version: null
+version: 0.0.0
```

Comment:
For my understanding, why not using a real version number, and just update it before release and adding a tag ?

Reply:
Because of the release process where tag is created on a release branch. It means 2 things:

- every latest release would require a commit back to devel to resync version
- In case of bug fix in another branch, it will require to also change version or it will create a messy situation.

So if we want to make real version, then it is easier to switch back to manual management and update it during the release process

Reply:
Thanks !